templates/patterns/page_header/pattern-page-header.html.twig

```diff
@@ -47,6 +47,8 @@
   'breadcrumb': _breadcrumb,
   'meta': _meta,
   'infos': _infos,
+  'background_image': ecl_branding == 'core' and background_image_url is not empty,
+  'background_image_url': background_image_url|default(''),
```

Comment:
```suggestion
  'background_image_url': background_image_url,
```
Let's remember that Twig does this for us, unless they hard-check it in the ECL component.